homeassistant/helpers/service.py

```diff
@@ -112,3 +116,46 @@ def extract_entity_ids(hass, service_call, expand_group=True):
             return [service_ent_id]
 
         return service_ent_id
+
+
+@bind_hass
+def get_description(hass, domain, service):
+    """Return the description (i.e. user documentation) for a service call."""
+
+    if domain == ha.DOMAIN:
+        import homeassistant.components as components
+        file = components.__file__
```

Comment:
`file` is a python builtin. Please use another variable name, eg `fil`, `comp_file` or similar.